### PR TITLE
Summary from dev to master

### PR DESCRIPTION
**Pull request by qik-in-mate**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        title: this.title,
+        subtitle: this.subtitle,
+        isShowSubtitle: false,
+        isFavorite: false
+      });
+      this.title = '';
+      this.subtitle = '';
+    },
+    addToFavorite(card) {
+      if (this.inFavorites.includes(card.id) === true) {
```
Comment (carrot-e) [inline index.js]: `if (this.inFavorites.includes(card.id)) {` -- does the same.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        return;
+      }
+      this.inFavorites.push(card.id);
+      card.isFavorite = true;
+    },
+    deleteFromFavorite(card) {
+      if (this.inFavorites.includes(card.id) === false) {
```
Comment (carrot-e) [inline index.js]: `if (!this.inFavorites.includes(card.id)) {`

**Review by carrot-e — COMMENTED** [review]
Well done.
Find my comments in the code.
I've found a bug:
Screenshot from 2019-04-23 10-48-38

When you delete cards using "remove first card" and "remove last card" buttons, the favorites counter is not updated.

This happens because you manually update `inFavorites` property to keep track of favorited cards.
To solve the problem of auto-updating, you can convert `inFavorites` to a **computed** property.

File: index.js (hunk 2 above)
Comment (qik-in-mate) [reply index.js]: I know. Thanks. It's easier for me to read the code.

File: index.js (hunk 1 above)
Comment (qik-in-mate) [reply index.js]: I know. Thanks. It's easier for me to read the code.

**Pull request merged**